<?xml version="1.0"?>
<launch>

  <!-- Launch catkinpkg_framework node -->
  <node pkg="dynamixel_controller" name="dynamixel_controller_node"
        type="dynamixel_controller_node" clear_params="true" output="screen" />

  <!-- Launch rviz -->
  <!--<node name="rviz_node" pkg="rviz" type="rviz" /-->

  <!-- Launch rqt -->
  <node pkg="rqt_gui" type="rqt_gui" name="rqt_gui" args="--perspective-file $(find dynamixel_controller)/rqt/dyn_reconfig.perspective" />

  <!-- Include rosparam yaml files -->
  <rosparam command="load" file="$(find dynamixel_controller)/parameter/parameter_bag.yaml" />

  <!-- Include other files -->
  <!--param name="object_files_path" type="str" value="$(find dynamixel_controller)/object_files.txt" /-->

</launch>
